xchange-getbtc/src/main/java/org/knowm/xchange/getbtc/dto/GetbtcException.java

```diff
@@ -0,0 +1,18 @@
+package org.knowm.xchange.getbtc.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class GetbtcException extends RuntimeException {
```

Comment:
if you want this to be thrown if the API returns HTTP error code or cannot deserialize the response then you have to add it to the throws declaration in each method in Getbtc and GetbtcAuthenticated